client/css/style.css

```diff
@@ -1492,18 +1504,16 @@ kbd {
 
 .context-menu-divider {
 	height: 1px;
-	margin: 6px 0;
 	background-color: rgba(0, 0, 0, .1);
 }
 
 .context-menu-item,
 .textcomplete-item {
 	cursor: pointer;
 	display: block;
-	padding: 4px 8px;
+	padding: 10px 8px;
 	color: #333;
-	margin-top: 6px;
-	margin-bottom: 6px;
```

Comment:
Actually, this has a tiny issue, in which it adds extra margin around elements within the same "group":

Before your PR | After your PR
--- | ---
<img width="186" alt="screen shot 2017-06-12 at 01 36 33" src="https://user-images.githubusercontent.com/113730/27020535-3628e5d2-4f10-11e7-8ea4-7190ff3fe4b2.png"> | <img width="183" alt="screen shot 2017-06-12 at 01 37 04" src="https://user-images.githubusercontent.com/113730/27020534-36277f08-4f10-11e7-908e-8dbbb2ef24e7.png">

Mind fixing this please?